### ISSUE TITLE
[color contrast] Passes when `color` & `background-color` is the same

### ISSUE THREAD

**Issue by Malvoz** [dequelabs/axe-core#1975]
Using the Chrome extension v4.2.0 (axe-core 3.4.0), color contrast isn't failing when foreground and background is the same color.

Reproduce: https://bird-direction.glitch.me/

**Comment by WilcoFiers** [comment]
Strictly speaking having a text with the same foreground and background color isn't a contrast issue under WCAG. I would strongly suggest not doing this, as it can mess with custom stylesheets (such as you can make with Stylish). Axe-core calls this out for review.
